datajoint/autopopulate.py

```diff
@@ -118,12 +122,12 @@ def populate(self, *restrictions, suppress_errors=False, reserve_jobs=False, ord
                             jobs.complete(self.target.table_name, key)
         return error_list
 
-    def progress(self, restriction=None, display=True):
+    def progress(self, *restrictions, display=True):
```

Comment:
I'd use the keyword arguments in a different order for clarity, but it would be fine with me. I think we should either allow `*restrictions` and `**kwrestrictions` or neither and have a simple `restrictions` argument.